src/Bicep.Core/Semantics/Namespaces/ThirdPartyNamespaceType.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Collections.Immutable;
+using Bicep.Core.TypeSystem;
+using Bicep.Core.TypeSystem.Providers;
+using Bicep.Core.TypeSystem.Providers.ThirdParty;
+using Bicep.Core.TypeSystem.Types;
+
+namespace Bicep.Core.Semantics.Namespaces
+{
+    public static class ThirdPartyNamespaceType
+    {
+        // harsh - we may not need a built in name or may have to get this from Create function 
+        public const string BuiltInName = "thirdparty";
+
+        public static NamespaceSettings Settings { get; } = new(
+            IsSingleton: true,
+            BicepProviderName: BuiltInName,
+            ConfigurationType: null,
+            ArmTemplateProviderName: "",
+            ArmTemplateProviderVersion: "");
```

Comment:
As you mentioned in your comment above:
`BicepProviderName`, `ArmTemplateProviderName`, `ArmTemplateProviderVersion` will all have to come from the `Create` method